main.go

```diff
@@ -43,7 +43,6 @@ func main() {
 	providerStore := flag.String("provider-store", "", "A non-default provider store to use, either \"none\" or \"dynamodb://table=<string>,ttl=<ttl-in-seconds>,queryLimit=<int>\"")
 	httpAPIAddr := flag.String("httpapi-addr", defaultHTTPAPIAddr, "Specify an IP and port to run the HTTP API server on")
 	delegateTimeout := flag.Int("delegate-timeout", 0, "Timeout for delegated routing in milliseconds")
-	reframeAddr := flag.String("reframe-addr", "", "Reframe API endpoint for delegated routing")
```

Comment:
are we getting the delegated content routing server from config now?
would be good to have a way to set that explicitly for hydras - in case we want to over-ride it to a specific load balancer / instance.

Reply:
It's set via the `provider-store` flag now, see https://github.com/libp2p/hydra-booster/commit/8be14bf61bff477319273e5757da95c382c01a73#diff-88c7467ead74488d50c14a1194946c7df31713d0146f0038b761a020551172b5R279-R284